vendor/github.com/kubev2v/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -460,6 +455,28 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 		// We'll get called again later when the data source exists
 		return nil
 	}
+	var rawBlock bool
+	if nil != pvc.Spec.VolumeMode && corev1.PersistentVolumeBlock == *pvc.Spec.VolumeMode {
+		rawBlock = true
+	}
+
+	// Calculate the args for the populator pod
+	var args []string
+	args, err = c.populatorArgs(rawBlock, unstructured)
+	if err != nil {
+		return err
+	}
+
+	// Overriding the namespace to our target namespace
+	// Taking the secret name from the args
```

Comment:
I don't think the comment about the secrets adds any value - the code is self-explanatory